arkane/common.py

```diff
@@ -252,7 +252,8 @@ def load_yaml(self, path, species, pdep=False):
             raise KeyError("Can only make objects if the `class` attribute in the dictionary is known")
         if class_name != 'ArkaneSpecies':
             raise KeyError("Expected a ArkaneSpecies object, but got {0}".format(class_name))
-        del data['class']
+        data = {key: data[key] for key in data.iterkeys() if key != 'class'}
```

Comment:
What's the difference between this and the deleted line?

Reply:
The deleted line deletes class from the original object, which changes data. In this file I realize now that this isn't an issue, and I can revert this back (it might be faster to use `del data['class']`).

I had used code like this in other files, where changing `data` could cause problems under certain circumstances. For example, if you were scripting and had defined `data` externally but passed it to functions like `recursive_make_dictionaries` multiple times.

Let me know what your preference on this is, I am happy to leave it in or revert this line.

Reply:
I think the original like is simpler. If it's not problematic any more, lets revert the change.
Same for `imaginary_frequency` below?